docs/_guide/03-styling-templates.md

```diff
@@ -0,0 +1,94 @@
+---
+layout: post
+title: Styling templates
+slug: styling-templates
+---
+
+lit-html just renders HTML; you can style HTML created with lit-html like any other HTML, using CSS. How you apply styles to lit-html templates depends on how you're using it—for example, if you're using lit-html inside a component system like LitElement, you can follow the patterns used by that component system.
+
+In most cases you should **not** use lit-html bindings inside a style sheet. It's more efficient to use lit-html bindings to manipulate the `class` and `style` attributes.  
+
+lit-html provides two directives for manipulating an element's `class` and `style` attributes:
```

Comment:
I think styleMap and classMap deserve a heading here. Maybe: "Styling individual Elements with classMap and styleMap". An example could be useful too.

Reply:
Yeah, I hated to add a section here because it felt really repetitive with the section in the reference. 

That example is already incredibly stupid, and I don't really want to repeat it ... Would love suggestions for better examples here.

Reply:
https://github.com/katejeffreys/classmaps/blob/master/index.js has a couple i was playing with